notebooks/hello-universe/Regressing_TESS_rotation_periods_with_CNNs/Regressing_TESS_rotation_periods_with_CNNs.ipynb

```diff
@@ -0,0 +1,1019 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sigmas = preds[:, 1]*test_dataset.pmax\n",
+    "frac = sigmas/pred_periods\n",
+    "\n",
+    "plt.figure()\n",
+    "plt.hist(sigmas, bins=20)\n",
+    "plt.xlabel(\"Predicted Period Uncertainty (days)\")\n",
+    "\n",
+    "plt.figure()\n",
+    "plt.hist(frac, bins=20)\n",
+    "plt.xlabel(\"Fractional Period Uncertainty\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f68d6882-9de6-4e3f-a7b0-352a285eb94d",
+   "metadata": {},
+   "source": [
+    "By 20 epochs we start to see a dip forming at sigma/period ~ 0.5 (this becomes more pronounced the longer we train), so let's use that to filter the results, weeding out \"bad\" predictions."
```

Comment:
What makes you say "by 20 epochs"?  I don't see any plots or the like that demonstrate this.